src/app/app-routing.module.ts

```diff
@@ -97,6 +97,11 @@ const routes: Routes = [
     loadChildren: () =>
       import('./search/search.module').then((m) => m.SearchModule),
   },
+  {
+    path: ApplicationRoutes.twoFactor,
+    loadChildren: () =>
```

Comment:
A guard is required here, right? So no one can load this module without the required context